SwiftIsland.xcodeproj/project.pbxproj

```diff
@@ -66,6 +66,13 @@
 		2ADD086B22ADB01400A90D1E /* TestDataManager.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2ADD086A22ADB01400A90D1E /* TestDataManager.swift */; };
 		2AF63104226DE96D00509053 /* MapViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2AF63103226DE96D00509053 /* MapViewController.swift */; };
 		2AF63107226E5CA300509053 /* Area.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2AF63106226E5CA300509053 /* Area.swift */; };
+		CE84EA252343D63400C9C82E /* About.swift in Sources */ = {isa = PBXBuildFile; fileRef = CE84EA242343D63400C9C82E /* About.swift */; };
+		CE84EA282343DA9A00C9C82E /* AboutViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = CE84EA272343DA9A00C9C82E /* AboutViewController.swift */; };
+		CE84EA2A234783DF00C9C82E /* About_valid.json in Resources */ = {isa = PBXBuildFile; fileRef = CE84EA29234783DF00C9C82E /* About_valid.json */; };
+		CE84EA2C2347843E00C9C82E /* AboutTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = CE84EA2B2347843E00C9C82E /* AboutTests.swift */; };
+		CE84EA2E23478B2E00C9C82E /* About_valid.json in Resources */ = {isa = PBXBuildFile; fileRef = CE84EA29234783DF00C9C82E /* About_valid.json */; };
+		CE84EA2F23478B4000C9C82E /* TestDataManager.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2ADD086A22ADB01400A90D1E /* TestDataManager.swift */; };
```

Comment:
This file should not be added to the SwiftIsland target; it is for the test target only.